histomicstk/segmentation/label/trace_label.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import ctypes
+from trace_object_boundary import trace_object_boundary
+
+
+def trace_label(imLabel, Connectivity=8):
+
+    imLabel = np.ascontiguousarray(imLabel, dtype=ctypes.c_int)
+
+    output = trace_object_boundary(imLabel, Connectivity)
+
+    return output
```

Comment:
I don't think there is a need for trace_label anymore because trace_object_boundary is doing the same thing. So remove it.